docker/Dockerfile

```diff
@@ -27,7 +27,9 @@ RUN mkdir /var/run/clamav && \
 
 RUN sed -i 's/^Foreground .*$/Foreground true/g' /etc/clamav/clamd.conf && \
     echo "TCPSocket 3310" >> /etc/clamav/clamd.conf && \
-    sed -i 's/^Foreground .*$/Foreground true/g' /etc/clamav/freshclam.conf
+    sed -i 's/^Foreground .*$/Foreground true/g' /etc/clamav/freshclam.conf && \
+    sed -i 's/^Checks .*$/Checks 12/g' /etc/clamav/freshclam.conf && \
+    printf "# Proxy: http://egress.internal:8080\\n# HTTPProxyServer egress.internal\\n# HTTPProxyPort 8080" >> /etc/clamav/freshclam.conf
```

Comment:
This seems to only add the comments to the file, or am I missing something?

How big is the config file? Would it make sense to have a copy of the config we want in this repo and copy it over the default one instead of trying to modify it in-place?

Reply:
Yeah its meant to be commented out.

So freshclam adds those values to the default config when it starts up and detects it should add them. So by adding them and commenting them out it doesn't re-add them.